mvpa2/algorithms/hyperalignment.py

```diff
@@ -198,6 +198,10 @@ def train(self, datasets):
         """
         params = self.params            # for quicker access ;)
         ca = self.ca
+        # Check to make sure we get a list of datasets as input.
+        if not (isinstance(datasets, list) or isinstance(datasets, tuple)):
```

Comment:
Sounds reasonable. How do I create a np.ndarray() of datasets to test? Couple of ways I tried didn't work. @yarikoptic